### PR TITLE
'login to listen to full song' prompt for non-logged in visitors

### LINKED ISSUE #115: For non-authed visitors, prompt: "Log in to listen to full song"
We need a simple way to alert non-logged in users that they need to log in to listen to more than 45 second previews. Here's some mockups of what it could look like and how it could function.

Design: If user is not logged in and is listening to a song preview, a bar displays just above the player control area while listening to a 45 second song preview. The bar states "Log in to listen to full song". The bar disappears when 45 second playback ends or playback is turned off by user.

If easier to program, the bar could instead stay visible at all times to non-logged in users. Most important thing is just to get the message out there so that new users understand that the player is functioning as designed and they need to log in to hear full tracks.

loginprompt_blackbg

loginprompt_white

### PR DESCRIPTION
**Pull request by mcfiredrill**
line_20220105_103559

resolves https://github.com/resonatecoop/stream/issues/115

This is my first PR, I am completely new to choojs/tachyons (although I have used tailwind alot, its simliar).
Looking forward to scathing feedback! :)

### CODE REVIEW

**Comment by mcfiredrill** [comment]
Just a note, I wasn't able to implement this part yet. Perhaps in another PR.
> The bar disappears when 45 second playback ends or playback is turned off by user.

File: packages/player-component/index.js [hunk 1]
```diff
@@ -320,8 +320,13 @@ class Player extends Nanocomponent {
   }
 
   renderPlayer () {
+    const isAuthenticated = !!this.state.user.uid
+
     return html`
       <div class="player-component flex flex-column h-100">
+        ${!isAuthenticated
+          ? html`<span class="flex flex-row w-100 justify-center bb b--light-silver">Log in to listen to full song</span>`
```
Comment (mcfiredrill) [inline packages/player-component/index.js:328]: I'm not sure if this border bottom color is correct

Comment (auggod) [reply packages/player-component/index.js:328]: You can use the following classes list instead
```b--mid-gray b--mid-gray--light b--near-black--dark```

**Comment by hakanto** [comment]
Ready to merge? @auggod

File: packages/player-component/index.js (hunk 1 above)
Comment (peterklingelhofer) [reply packages/player-component/index.js:328]: Here are the three options @auggod suggested above. Doesn't seem to make much a difference in dark mode on my system, but in light mode note the differences with the prompt's top horizontal border outline. Here's how they're all showing up for me:

### `b--mid-gray`
b--mid-gray

```suggestion
          ? html`<span class="flex flex-row w-100 justify-center bb b--mid-gray">Log in to listen to full song</span>`
```

### `b--mid-gray--light`
b--mid-gray--light

```suggestion
          ? html`<span class="flex flex-row w-100 justify-center bb b--mid-gray--light">Log in to listen to full song</span>`
```

### `b--near-black--dark`
b--near-black--dark

```suggestion
          ? html`<span class="flex flex-row w-100 justify-center bb b--near-black--dark">Log in to listen to full song</span>`
```

Comment (peterklingelhofer) [reply packages/player-component/index.js:328]: The 'b--mid-gray' (the first option above) seems to make the most sense in the UI to my eyes. Once that change is made this should be ready to merge.

**Pull request merged**